cern.root.ROOT.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
This seems like a primarily a console application that can sometimes open windows for plots etc

The main desktop file and command both opens a terminal. This should be marked as a console-application then

Reply:
I modified it after seeing some details